Doc/library/exceptions.rst

```diff
@@ -204,18 +204,24 @@ The following exceptions are the exceptions that are usually raised.
    assignment fails.  (When an object does not support attribute references or
    attribute assignments at all, :exc:`TypeError` is raised.)
 
-   The :attr:`name` and :attr:`obj` attributes can be set using keyword-only
-   arguments to the constructor. When set they represent the name of the attribute
-   that was attempted to be accessed and the object that was accessed for said
-   attribute, respectively.
+   The optional *name* and *obj* keyword-only arguments
+   set the corresponding attributes:
+
+   .. attribute:: name
+
+      The name of the attribute that was attempted to be accessed.
+
+   .. attribute:: obj
+
+      The object that was accessed for said attribute.
 
    .. versionchanged:: 3.10
       Added the :attr:`name` and :attr:`obj` attributes.
 
 .. exception:: EOFError
 
    Raised when the :func:`input` function hits an end-of-file condition (EOF)
-   without reading any data. (N.B.: the :meth:`io.IOBase.read` and
+   without reading any data. (N.B.: the :meth:`!io.IOBase.read` and
```

Comment:
Let's also replace the Latin abbreviation: https://devguide.python.org/documentation/style-guide/#use-simple-language
```suggestion
   without reading any data. (Note: the :meth:`!io.IOBase.read` and
```